core/README.rst

```diff
@@ -3,75 +3,47 @@ Toga
 
 A Python native, OS native GUI toolkit.
 
-Quickstart
-----------
-
-To get a demonstration of the capabilities of Toga, run the following::
-
-    $ pip install toga-demo
-    $ toga-demo
-
-This will pop up a GUI window with some sample widgets.
-
-Prerequisites
-~~~~~~~~~~~~~
-
-Toga has some minimum requirements:
-
-* If you're on OS X, you need to be on 10.10 (Yosemite) or newer.
-
-* If you're on Linux, you need to have GTK+ 3.4 or later. This is the version
-  that ships starting with Ubuntu 12.04 and Fedora 17.
-
-* If you want to use the WebView widget, you'll also need to have WebKit, plus
-  the GI bindings to WebKit installed.
+This package provides the core Toga API. In order to use Toga, you'll also need to
+install a backend that implements the core Toga API for that platform:
 
-    * For Ubuntu that's provided by the (``libwebkitgtk-3.0-0``) and
-      (``gir1.2-webkit-3.0``) packages.
+* **Android** `toga-android <https://pypi.org/project/toga-android>`__
+* **iOS** `toga-iOS <https://pypi.org/project/toga-iOS>`__
+* **Linux** `toga-gtk <https://pypi.org/project/toga-gtk>`__
+* **macOS** `toga-cocoa <https://pypi.org/project/toga-cocoa>`__
+* **Web** `toga-web <https://pypi.org/project/toga-web>`__
+* **Windows** `toga-winforms <https://pypi.org/project/toga-winforms>`__
 
-    * For Fedora it's all provided in the (``webkitgtk3``) package.
+Minimum requirements
+--------------------
 
-If these requirements aren't met, Toga either won't work at all, or won't have
-full functionality.
+* Toga requires **Python 3.8** or newer. Python 2 is not supported.
 
+* If you're on macOS, you need to be on 10.10 (Yosemite) or newer.
 
-Problems with source installs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* If you're on Windows, you'll need Windows 10 or newer. If you are using
+  Windows 10 and want to use a WebView to display web content, you will also
```

Comment:
Duplicating all of this is just giving ourselves extra work to keep it in sync. Suggest replacing it all with:
```
* For platform-specific requirements, see the READMEs of each backend.

Reply:
Good point - however, we also need to make sure we consistently link to the actual place where these requirements are specified so we have a single source of truth. I'll take another pass at this.